.github/workflows/automatic-pr-update.yml

```
@@ -20,6 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: rasahq/merge-pal-action@master
+      - uses: rasahq/update-pr-branch@601ac3dadf708b14e7f0252d98230538235f99d9
+        if: github.repository == 'RasaHQ/rasa'
         with:
           token: ${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN || secrets.GITHUB_TOKEN}}
```

One more thing, documentation says action only works with [PAT](https://github.com/rasahq/update-pr-branch#token), we can remove `secrets.GITHUB_TOKEN`

```suggestion
          token: ${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN }}
```